qa/workunits/cephadm/nvme_initiator.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -ex
+
+sudo modprobe nvme-fabrics
+sudo modprobe nvme-tcp
+
+GATEWAY=$(cat /etc/ceph/nvmeof.cfg |grep 'gateway_addr' | awk -F'[, ]' '{print $3}')
+IP=$(cat /etc/ceph/nvmeof.cfg |grep 'ip_address' | awk -F'[, ]' '{print $3}')
```

Comment:
If you're already writing one file, why not also write an `nvmeof.env` that you can `source` instead of doing complex grep/awk tricks? It also insulates you from any future changes to the config file content.

Reply:
I got the idea from https://github.com/ceph/ceph/blob/6ada314d1c4170114c2eba1ade4d7319e796b3d6/qa/tasks/cephadm.py#L83

Using `.env` sounds like a good idea, I'll make the env variable names more explicit so they don't override anything important. Thank you!